src/features/productDetails/ProductDetails.tsx

```diff
@@ -38,10 +46,27 @@ const ProductDetails = () => {
                         <h2 className="text-3xl">{product?.title}</h2>
                         <h2 className="text-2xl">{formattedPrice(product?.price ?? 0)}</h2>
```

Comment:
Consider using a simple \<p> tag rather than the \<h2> tag because the latter is a semantic tag which is usually used for subheadings of articles/posts (not necessarily appropriate here)